api/backends/index.js

```diff
@@ -4,25 +4,43 @@ import axios from 'axios'
 export let visualisationApiUrl = process.env.visualisationApiUrl
 export let dssauthUrl = process.env.dssauthUrl
 export let discoveryApiUrl = process.env.discoveryApiUrl
+export let nivsMinioUrl = process.env.nivsMinioUrl
+export let nidMinioUrl = process.env.nidMinioUrl
 export let environment = process.env.environment
+export let instanceID = process.env.INSTANCE_ID
 
 const appMode = process.env.NODE_ENV
+
+function useDefaultAPIUrls() {
+  visualisationApiUrl =
+    'https://dafni-nivs-api-review-dev-4jxwt5.staging.dafni.rl.ac.uk'
+  dssauthUrl =
+    'https://dafni-dss-dssauth-review-dev-o2yn5p.staging.dafni.rl.ac.uk'
+  discoveryApiUrl =
+    'https://dafni-search-and-discovery-api-review-dev-4jxwt5.staging.dafni.rl.ac.uk'
+  nivsMinioUrl = null
+  nidMinioUrl = null
+  instanceID = 'b57525e7-0a5a-43da-b489-741c18c9f6d8'
+}
+
 if (appMode !== 'production') {
-  visualisationApiUrl = "https://dafni-nivs-api-review-dev-4jxwt5.staging.dafni.rl.ac.uk"
-  dssauthUrl = "https://dafni-dss-dssauth-review-dev-o2yn5p.staging.dafni.rl.ac.uk"
-  discoveryApiUrl = "https://dafni-search-and-discovery-api-review-dev-4jxwt5.staging.dafni.rl.ac.uk"
+  useDefaultAPIUrls()
   environment = "dev"
 }
 
 async function backends() {
   try {
-    const response = await axios.get('/backends/backends.json')
+    const response = await axios.get('./backends/backends.json')
     visualisationApiUrl = response.data.visualisationApiUrl
     dssauthUrl = response.data.dssauthUrl
     discoveryApiUrl = response.data.discoveryApiUrl
+    nivsMinioUrl = response.data.nivsMinioUrl
+    nidMinioUrl = response.data.nidMinioUrl
     environment = response.data.node_env
+    instanceID = response.data.instanceID
   } catch (error) {
-    console.error('Error while loading settings from server' + error)
+    console.error(`Error while loading settings from server: ${error}`)
+    useDefaultAPIUrls()
```

Comment:
How do the Minio URLs get set if we go down this path? Or is it a case of them not needing to be set?

Reply:
Just seen that I _think_ they get replaced?

If so, might be worth adding a comment to the nulls above just highlighting that?

Reply:
useDefaultAPIUrls() sets all the variables to the right thing. The minio urls are variables that actually contain the proxy urls to replace the true minio urls with (in truth its only the hostname not the full url). There is only a cors proxy for staging / production so we only need the urls to be non-null in staging / prod

Reply:
The catch _should_ only happen locally

Reply:
Makes sense, thanks for the comment.